### ISSUE TITLE
Copying over all annotations in correction and automation mode

### ISSUE THREAD

**Issue by nkrot** [webanno/webanno#495]
This is a feature request.

It may happen that the suggested annotations (in correction and automation mode) are entirely correct or mostly correct. Therefore it would speed up annotation process if the users are given a possibility to just copy over with a single click all annotations for a selected sentence to its counterpart in the annotation area.

In case the sentence in the annotation area already carries some annotations, they can be merged and highlighted as in the curation mode.

A more sophisticated scenario is copying over all annotations of *remaining* words (remaining means after the last annotated word that happens to have the "focused" status). Say, I have gone though a part of a sentence transferring annotations the usual way by clicking them individually, then I quickly review the rest of the sentence and see that all remaining tokens are annotated correctly. At this point I would like to transfer remaining annotations with a single click.

Does any of the above make sense and is useful?

**Comment by seyyaw** [comment]
@nkrot This was one of the feature in WebAnno  requested many times. But we decided that coping suggestion annotations without user action (a click here) to the annotation pane has a bad consequence *regarding annotation quality). But, if the project admin believes that the suggestion annotations are good enough, WebAnno allows exporting the suggestion annotations instead.

**Comment by nkrot** [comment]
I am not suggesting "copying suggestion annotations without user action". The user still has to take an action. My suggestions is rather about taking **less** actions than there are individual annotations

**Comment by seyyaw** [comment]
May be we can make this configurable, that means the admin could specifically enable such features from the project setting page.

**Comment by nkrot** [comment]
I agree, the project manager should decide if s/he trusts the annotators enough to allow them to transfer suggestions. As a project manager I would disallow this possibility at an initial stage of the annotation project but enable it at a later step.
